tachyon/math/polynomials/univariate/radix2_evaluation_domain_unittest.cc

```diff
@@ -25,7 +25,7 @@ TYPED_TEST_SUITE(Radix2EvaluationDomainTest, PrimeFieldTypes);
 
 TYPED_TEST(Radix2EvaluationDomainTest, FFTBatch) {
   using F = TypeParam;
-  for (uint32_t log_r = 0; log_r < 5; ++log_r) {
+  for (uint32_t log_r = 1; log_r < 5; ++log_r) {
```

Comment:
Why did you change this line?

Reply:
It was due to edge case in FFTBatch. I will change it and use `EXPECT_DEATH` here.